contracts/EASResolver/EASResolver.sol

```diff
@@ -0,0 +1,54 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.7;
+
+import { SchemaResolver } from "./SchemaResolver.sol";
+
+import { IEAS, Attestation } from "./IEAS.sol";
+
+contract RecipientResolver is SchemaResolver {

```

Comment:
Rename contract to WhitelistResolver. Change the filename to match the contract name.